analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyNullableValueAccessTest.cs

```diff
@@ -61,6 +61,10 @@ public class EmptyNullableValueAccessTest
             .WithOptions(ParseOptionsHelper.FromCSharp8)
             .Verify();
 
+    [TestMethod]
+    public void EmptyNullableValueAccess_Roslyn_NullableContext() =>
+        roslynCS.AddPaths("EmptyNullableValueAccess.NullableContext.cs").WithOptions(ParseOptionsHelper.FromCSharp8).WithConcurrentAnalysis(false).Verify();
```

Comment:
I think the trick is to add an empty newline at the beginning of the test file, but I'm not sure. Please give it a try.
```suggestion
        roslynCS.AddPaths("EmptyNullableValueAccess.NullableContext.cs").WithOptions(ParseOptionsHelper.FromCSharp8).Verify();
```

Reply:
It does work without `WithConcurrentAnalysis(false)`, even not adding the empty newline at the beginning of the test file.

So, I have removed `WithConcurrentAnalysis(false)` in https://github.com/SonarSource/sonar-dotnet/pull/7006 (new PR for all issues detected on this PR but on code already merged into `feature/SE`).